test/system/252-quadlet.bats

```diff
@@ -1412,4 +1412,129 @@ EOF
     run_podman rmi --ignore $(pause_image)
 }
 
+@test "quadlet - pod simple" {
```

Comment:
`.pod` files are not supported in 4.8.x. This test should not be added

Reply:
done. PTAL